apps/wallet/src/background/accounts/zklogin/ZkLoginAccount.ts

```diff
@@ -282,12 +342,66 @@ export class ZkLoginAccount
 			randomness: randomness.toString(),
 			jwt,
 		};
-		ephemeralValue[serializeNetwork(activeNetwork)] = credentialsData;
-		await this.setEphemeralValue(ephemeralValue);
+		await this.saveCredentialsData(credentialsData);
```

Comment:
extracting to a method so we can update credentials for the other address if it exists